strings/splash.json

```diff
@@ -0,0 +1,26 @@
+{
+	"splashes": [
+		"Paved by the tears of coders.",
+		"Built on the only engine 100% of people find unusable!",
+		"It's a miracle BYOND hasn't crashed yet.",
+		"The only 'popular' game on BYOND.",
```

Comment:
We probably don't want to get on the bad side of byond. I'd remove these ones.
The second one can be replaced with "It's a mircale it(/we) hasn't crashed yet"